katalog/tests/istio-operator/requirements/Furyfile.yml

```diff
@@ -3,9 +3,11 @@
 # license that can be found in the LICENSE file.
 
 versions:
-  monitoring: v1.11.1
-  logging: v1.7.1
+  monitoring: v2.0.0
+  logging: v3.1.0
+  ingress: v1.13.0
```

Comment:
there are newer versions of this modules, not a blocking thing for this PR but would be better to test against the latest releases